### ISSUE TITLE
Function question

### ISSUE THREAD

**Issue by amymhaddad** [amymhaddad/exercises_for_programmers#97]
![screen shot 2018-09-12 at 8 45 16 am](https://user-images.githubusercontent.com/39930340/45425721-395c3b80-b668-11e8-80fd-f68bcdd34875.png)

The program should ask the user to enter the balance of a credit card and the APR of the card. The program should then return the number of months needed. Use a function called calculateMonthsUntilPaidOff, which accepts the balance, the APR, and the monthly payment as its arguments and returns the number of months. Don’t access any of these values outside the function.

**Questions:**
-I'm not sure what I should use as parameters here? So I use ```*months_to_pay_cc``` to allow an infinite number of arguments to be passed. Is this correct? 

-The problem stated not to get user input outside of the function, which is why I include it inside of it. However, this seems confusing with the arguments (the problem states what to include as arguments). 

-I can't get the calculation to run properly, which is why I divided it into several lines. But it still doesn't work. What's going on?!

```
import math 

def calc_months_until_paid_off(*months_to_pay_cc):
    """Write a function to determine the number of months until credit card is paid off"""

    # Get input from user
    credit_card_balance = int(input("Enter your credit card balance: $"))
    apr = int(input("Enter the APR as a percent: "))
    monthly_payment = int(input("Enter your monthly payment: $"))

    # Calculate apr as a decimal and the apr daily rate
    apr_as_decimal = apr / 100
    apr_daily_rate = apr_as_decimal / 365

    # Calcuate number of months to pay off credit card
    top_of_calculation = math.log(1 + credit_card_balance / monthly_payment (1 - (1 - apr_daily_rate) ** 30))
    bottom_of_calculation = math.log(1 + apr_daily_rate)
    months_to_pay_cc = -1/30 * top_of_calculation / bottom_of_calculation

    return(months_to_pay_cc)

calc_months_until_paid_off(credit_card_balance, apr, monthly_payment)

# print statement
```

**Comment by paulghaddad** [comment]
@amymhaddad So the input to this function appears to be the four bullets in your image above. These are the things the function needs to do it's work. So to move on, you should have exactly four parameters. I'd ask the user for input on all four parameters, and _then_ call this function, passing in those values as arguments. Does that make sense? 

When thinking about parameters, ask yourself what is the input this function needs, and that's your answer.

**Comment by amymhaddad** [comment]
@paulghaddad Yes. I was confused b/c in Eric Matthes book he distinguishes between parameters and arguments, whereas in ex's for programmers they use the terms interchangably. All set.
